lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -0,0 +1,16 @@
+module SqlFootprint
+  class SqlAnonymizer
+    GSUBS = {
+      /\s+IN\s+\(.*\)/ => ' IN (values-redacted)'.freeze, # IN clauses
+      /\s+[0-9]+/ => ' number-redacted'.freeze, # numbers
```

Comment:
I was anchoring w/ the '=' (and now just operator list) because tables and columns can be named as numbers.